token-lending/program/src/processor.rs

```diff
@@ -419,6 +419,26 @@ fn _refresh_reserve<'a>(
     Ok(())
 }
 
+/// Lite version of refresh_reserve that should be used when the oracle price doesn't need to be updated
+/// BE CAREFUL WHEN USING THIS
+fn _refresh_reserve_lite<'a>(
+    program_id: &Pubkey,
+    reserve_info: &AccountInfo<'a>,
+    clock: &Clock,
+) -> ProgramResult {
+    let mut reserve = Reserve::unpack(&reserve_info.data.borrow())?;
+    if reserve_info.owner != program_id {
```

Comment:
Should we also check that the reserve belongs to the same lending market?

Reply:
I guess probably doesn't need since the original refresh_reserve also doesn't. That one instead checks that Pyth and Switchboard oracles match, which I guess isn't needed here since it doesn't actually use the feeds.